microservice/src/main/java/nearsoft/skytouch/microservice/producer/ProductProducer.java

```diff
@@ -0,0 +1,8 @@
+package nearsoft.skytouch.microservice.producer;
+
+import nearsoft.skytouch.microservice.channel.ProductChannel;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+
+@EnableBinding(ProductChannel.class)
+public class ProductProducer {
```

Comment:
Is this class needed?

Reply:
Yes to make use of the ProductChannel class